conjure-undertow-processor/src/main/java/com/palantir/conjure/java/undertow/processor/generate/ConjureUndertowEndpointsGenerator.java

```diff
@@ -388,24 +389,18 @@ public Void cookie(
             }
 
             @Override
-            public Void authCookie(String variableName, String cookieName, String deserializerFieldName) {
-                TypeName paramType = def.argType().match(ArgTypeTypeName.INSTANCE);
-                additionalFields.add(ImmutableAdditionalField.builder()
-                        .field(FieldSpec.builder(
-                                        ParameterizedTypeName.get(ClassName.get(Deserializer.class), paramType.box()),
-                                        deserializerFieldName,
-                                        Modifier.PRIVATE,
-                                        Modifier.FINAL)
-                                .build())
-                        .constructorInitializer(CodeBlock.builder()
-                                .addStatement(
-                                        "this.$N = new $T(runtime, $S)",
-                                        deserializerFieldName,
-                                        BearerTokenCookieDeserializer.class,
-                                        cookieName)
-                                .build())
-                        .build());
-                handlerBuilder.addStatement("$T $N = $L", paramType, variableName, invokeDeserializer(def));
```

Comment:
The indirection through `BearerTokenCookieDeserializer` seems unnecessary and is inconsistent with the code generated for `AuthHeader`, where we invoke `runtime.auth()` directly.